pkgs/tools/inputmethods/fusuma/Gemfile

```diff
@@ -1,2 +1,2 @@
 source 'https://rubygems.org'
-gem "fusuma"
+gem 'fusuma', '2.4.1'
```

Comment:
Is that working with the updater?

Reply:
Did you mean `nixpkgs-update` ? I tried it both from `NixOS/nixpkgs` master branch (current) and my fork (PR) gave the same error:
```
> nixpkgs-update update "fusuma 1.3.0 2.4.1"
nixpkgs-update: Maybe.fromJust: Nothing
CallStack (from HasCallStack):
  error, called at libraries/base/Data/Maybe.hs:148:21 in base:Data.Maybe
  fromJust, called at src/Utils.hs:306:12 in nixpkgs-update-0.4.0-1LZpx5N2l6q8e22qNSkACg:Utils
```

Reply:
if you mean running `bundler update` it wont update because it's hardcoded, I'll rever the `Gemfile`